lake/api/api.go

```diff
@@ -36,6 +37,10 @@ type Interface interface {
 	UpdateIndex(ctx context.Context, names []string, pool ksuid.KSUID, branchName string) (ksuid.KSUID, error)
 }
 
+func IsRemoteLake(u *storage.URI) bool {
```

Comment:
I don't think remote is the right term here since a lake on S3 is also remote in a sense. Maybe IsAPILake, IsHTTPLake, or IsRESTLake?

Reply:
Right. Perhaps sine we call it service everywhere `IsLakeService`?

Reply:
That's perfect!